make_queue/models/models.py

```diff
@@ -208,8 +208,21 @@ def validate(self):
             return False
 
         # Check if the user can make the given reservation/edit
+        return self.quota_can_make_reservation()
+
+    # Check if the start time is before current time
+    def reservation_starts_before_now(self):
+        return self.start_time < timezone.now()
+
+    # Check if start time is after end time
```

Comment:
This comment should be a docstring instead. Generally, when a comment describes a function or class, it should be a docstring. You can make it like this:
```python
def check_start_time_after_end_time(self):
    """Check if start time is after end time"""
```

More info on docstrings: https://www.python.org/dev/peps/pep-0257/